spec/codecs/multiline_spec.rb

```diff
@@ -157,4 +157,52 @@
 
     end
   end
+
+  context "with non closed multiline events" do
+    let(:random_number_of_events) { rand(300..1000) }
+    let(:sample_event) { "- Sample event" }
+    let(:events) { decode_events }
+
```

Comment:
if you are generating random number of event, maybe you could loop these tests like10 times, so that you get the benefit of randomizing the number of events by running it multiple times without the same parameters. if it's too slow, just lower the random number of events and the max lines/bytes ?